### ISSUE TITLE
Read role is required with AAA and no-cache strat on volumedriver

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/framework#789]
We've tried to setup a vPool with Alba Accelerated Alba as READ cache. With volumedriver strategy `no-cache`. But we still need a READ role as seen here: 

```
2016-08-09 11:46:54 21500 +0200 - ctrl-stras-03 - 169456/139948666689344 - celery/celery.worker.job - 295 - ERROR - Task ovs.storagerouter.add_vpool[eb1ac90a-da48-4296-8bd1-20d7ddb23214] raised unexpected: ValueError('Errors validating the partition roles:\n - At least 1 READ partition role is required',)
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/celery/app/trace.py", line 240, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/celery/app/trace.py", line 438, in __protected_call__
    return self.run(*args, **kwargs)
  File "/opt/OpenvStorage/ovs/lib/storagerouter.py", line 532, in add_vpool
    raise ValueError('Errors validating the partition roles:\n - {0}'.format('\n - '.join(set(error_messages))))
ValueError: Errors validating the partition roles:
 - At least 1 READ partition role is required
```

**Comment by khenderick** [comment]
Seems it's indeed some leftover and apparently I validated all my changed with actually a READ role assigned to one of my partitions. However, the read partition itself won't be used in that case, so you can just - while awaiting this fix - assign it to whatever partition.

**Comment by khenderick** [comment]
Fixed by #989, packaged in openvstorage-2.7.3-rev.4076.66ee246

**Comment by JeffreyDevloo** [comment]
### Information
The issue described in the ticket is currently impossible to reproduce because of the READ role being removed since https://github.com/openvstorage/framework/pull/1130.
The main reason behind this abrupt closing as that we are far behind in validation...
Closing because it handles about a deprecated issue.

### Current Packages
- openvstorage                         2.7.6-rev.4323.a37c7f7-1        amd64        openvStorage
- openvstorage-backend                 1.7.6-rev.817.91abb7c-1         amd64        openvStorage Backend plugin
- openvstorage-backend-core            1.7.6-rev.817.91abb7c-1         amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps         1.7.6-rev.817.91abb7c-1         amd64        openvStorage Backend plugin Web Applications
- openvstorage-core                    2.7.6-rev.4323.a37c7f7-1        amd64        openvStorage core
- openvstorage-hc                      1.7.6-rev.817.91abb7c-1         amd64        openvStorage Backend plugin HyperConverged
- openvstorage-sdm                     1.6.6-rev.449.77bd347-1         amd64        Open vStorage Backend ASD Manager
- openvstorage-webapps                 2.7.6-rev.4323.a37c7f7-1        amd64        openvStorage Web Applications
